crates/sui/src/unit_tests/fixtures/upgrade_errors/all_v1/Move.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "upgrades"
+edition = "2024.beta" # edition = "legacy" to use legacy (pre-2024) Move
+
+[dependencies]
+Sui = { git = "https://github.com/MystenLabs/sui.git", subdir = "crates/sui-framework/packages/sui-framework", rev = "framework/testnet" }
```

Comment:
Better to use a "local" dep for `Sui` in tests -- otherwise these tests will be slower and more flaky because they involve a network call out to fetch the git repo.

Reply:
(ditto for all the other manifests)

Reply:
turns out dont I need the dep